_data/glossary/statefulset.yml

```diff
@@ -1,18 +1,18 @@
 id: statefulset
 name: StatefulSet
-formerly:
+aka:
 - PetSet
 related:
-- Deployment
-- Pod
+- deployment
+- pod
 tags:
-- Storage
-- Workload
-- API Object
+- core-object
+- workload
+- storage
 short-description: |
-  Manage the deployment and scaling of a set of Pods, *and provide guarantees about ordering*. They do so by maintaining a *unique*, sticky identity for each of their Pods.
+  Manages the deployment and scaling of a set of Pods, *and provides guarantees about the ordering and uniqueness* of these Pods.
```

Comment:
1. Remove italics.
2. Consistent caps: Pods/pods

Reply:
1. I think this is worth italicizing because this is chiefly what distinguishes StatefulSets from Deployments. Let me know if you strongly want to change it though.
2. Going to consistently capitalize as Pods (to distinguish it as a K8s object).